lib/modules/zfs.go

```diff
@@ -0,0 +1,60 @@
+package modules
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/davidscholberg/go-i3barjson"
+	"os/exec"
+	"strings"
+)
+
+// Zfs represents the configuration data for the ZFS block
+type Zfs struct {
+	BlockConfigBase `yaml:",inline"`
+	PoolName        string `yaml:"zpool_name"`
+	ZpoolBin        string `yaml:"zpool_bin"`
```

Comment:
I typically prefer to have an explicit path in the event that a non-standard binary name or obtuse path is involved. However, I'll defer to the implementation goals you have for the project. 'zpool' is now hard-coded in zfs.go, and needs to be in the users path.